contrib/IECoreUSD/src/IECoreUSD/USDScene.h

```diff
@@ -69,6 +69,11 @@ class USDScene : public IECoreScene::SceneInterface
 		bool hasTag( const Name &name, int filter ) const override;
 		void readTags( NameList &tags, int filter ) const override;
 		void writeTags( const NameList &tags ) override;
+
+		NameList setNames() const;
```

Comment:
We'll need to add `override` here to keep Clang happy.

Reply:
I offer 30e9a8f to gods of CLANG!